### PR TITLE
docs: upgrade 0.7.2

### PR DESCRIPTION
**Pull request by dl239**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

### CODE REVIEW

**Review by lumianph — APPROVED** [review]

**Review by lumianph — CHANGES_REQUESTED** [review]
please also review and merge those PRs:
#3066 
#3065 
#3034 
https://github.com/4paradigm/OpenMLDB/pull/3035
https://github.com/4paradigm/OpenMLDB/pull/3027
https://github.com/4paradigm/OpenMLDB/pull/3016

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,30 @@
 # Changelog
 
+## [0.7.2] - 2023-02-17
+
+### Features
```
Comment (aceforeverd) [inline CHANGELOG.md:5]: #3003 requires a breaking change notice ?

Comment (dl239) [reply CHANGELOG.md:5]: I do not think this is a breaking change: 
- user is no aware of this change
- new version is compatible with the old when upgrading

Comment (aceforeverd) [reply CHANGELOG.md:5]: Deploy response time will still face overflow issue from upgrading. At least a notable upgrade notice should exists in ChangeLog

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,30 @@
...
+- Support the new SQL clause `WITH` (#2846 @aceforeverd)
```
Comment (aceforeverd) [inline CHANGELOG.md]: The with clause feature should mark experimental.

Comment (lumianph) [reply CHANGELOG.md]: > The with clause feature should mark experimental.

let's mark it as [Alpha], please also do this in the doc

File: CHANGELOG.md (hunk 1 above)
Comment (dl239) [reply CHANGELOG.md:5]: ok

**Review by aceforeverd — APPROVED** [review]

**Review by lumianph — APPROVED** [review]

**Pull request merged**